src/App.tsx

```diff
@@ -1,51 +1,64 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
+
 import './App.scss';
+import { SortType } from './enums/sort-type.enum';
+import { Good } from './types/Good.type';
+import { goodsFromServer } from './data/goods-from-server';
+import { GoodList } from './components/GoodList';
+import { ButtonGroup } from './components/ButtonGroup';
+
+function getPreparedGoods(
+  goods: Good[],
+  sortField: SortType | '',
+  isReversed: boolean,
+): Good[] {
+  const preparedGoods = [...goods];
+
+  if (sortField) {
+    preparedGoods.sort((good1, good2) => {
+      switch (sortField) {
+        case SortType.ALPHABETICALLY:
+          return good1.localeCompare(good2);
+
+        case SortType.BY_LENGTH:
+          return good1.length - good2.length;
+
+        default:
+          return 0;
+      }
+    });
+  }
 
-export const goodsFromServer = [
-  'Dumplings',
-  'Carrot',
-  'Eggs',
-  'Ice cream',
-  'Apple',
-  'Bread',
-  'Fish',
-  'Honey',
-  'Jam',
-  'Garlic',
-];
+  if (isReversed) {
+    preparedGoods.reverse();
+  }
+
+  return preparedGoods;
+}
```

Comment:
The `getPreparedGoods` function name should start with a verb to indicate what the function does. Consider renaming it to something like `prepareGoods` to follow the [CODE STYLE] checklist item.

Reply:
The `getPreparedGoods` function name describes the essence of what the function does, so I would leave that name.